internal/xsql/collection.go

```diff
@@ -223,7 +223,11 @@ func (w *WindowTuples) Meta(key, table string) (interface{}, bool) {
 }
 
 func (w *WindowTuples) All(_ string) (map[string]interface{}, bool) {
-	return w.ToMap(), true
+	if len(w.Content) == 0 {
+		m := make(map[string]interface{})
+		return m, true
+	}
+	return w.Content[0].All("")
```

Comment:
Currently, select * for window will return all rows. Will this change that behavior?

Reply:
In the ToMap function, only w.Content[0] is being selected, so I don't think it will have any impact on the behavior.